### PR TITLE
Polls 06-14-2021

### PR DESCRIPTION
**Pull request by blimpa**
@0xElihu

(I can't add you as a proper reviewer for some reason)

### CODE REVIEW

File: governance/polls/Ratification Poll - Modify Core Unit Budget, RWF-001 (MIP40c3-SP11) - June 14, 2021.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Review
+
+The community can vote in this poll to express support or opposition to this proposal being accepted and implemented in the Maker Protocol.
+
+The proposal in its entirety [can be found here](https://github.com/makerdao/mips/blob/9a3f0e6842a27a38f0e7a56fc6cc1826b478f38b/MIP40/MIP40c3-Subproposals/MIP40c3-SP12.md). The sentence summary and paragraph summary have been provided by the MIP author:
+
+**Sentence Summary**
+
+*MIP40c3-SP12 supersedes the budget for the Real-World Finance Core Unit defined in (MIP40c3-SP1)[https://forum.makerdao.com/t/mip40c3-sp1-modify-core-unit-budget-real-world-finance/6225].*
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Modify Core Unit Budget, RWF-001 (MIP40c3-SP11) - June 14, 2021.md:30]: link not displaying properly

File: governance/polls/Ratification Poll - Modify Core Unit Budget, RWF-001 (MIP40c3-SP11) - June 14, 2021.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+**Paragraph Summary**
+
+*With (MIP40c3-SP1)[https://forum.makerdao.com/t/mip40c3-sp1-modify-core-unit-budget-real-world-finance/6225] the Real-World Finance team was able to onboard the first Real-World Asset and provide some financials for MakerDAO. Nevertheless, as no Legal Core Unit is expected in the short term, we need to increase the budget for legal expenses. We also need to solidify the data analytics part, which is currently limited, to integrate MakerDAO and RWA data to provide transparency and insights to stakeholders.*
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Modify Core Unit Budget, RWF-001 (MIP40c3-SP11) - June 14, 2021.md:34]: link not displaying properly

File: governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+title: Ratification Poll for Oracles Core Unit (ORA-001) Subproposal Set - June 14, 2021
+summary: Signal your support or opposition for these proposals being accepted and implemented in the Maker Protocol. 
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp12-adding-collateral-onboarding-core-unit/8037
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md:4]: wrong link here


correct one: https://forum.makerdao.com/tag/ora-001

File: governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to these proposals being accepted and implemented in the Maker Protocol.
+
+This Inclusion Poll MIP Set contains the following MIPs:
+
+* [MIP39c2-SP13: Adding Oracles Core Unit](https://github.com/makerdao/mips/blob/1a462c77d30bc603131b4f3c54a7486801395a7e/MIP39/MIP39c2-Subproposals/MIP39c2-SP3.md) | [Discussion Link](https://forum.makerdao.com/t/mip39c2-sp12-adding-collateral-onboarding-core-unit/8037)
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md:28]: Most of the links in the review section are messed up.

File: governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+* [MIP40c3-SP15: Modify Oracles Core Unit Budget](https://github.com/makerdao/mips/blob/1a462c77d30bc603131b4f3c54a7486801395a7e/MIP39/MIP39c2-Subproposals/MIP39c2-SP3.md) | [Discussion Link](https://forum.makerdao.com/t/mip40c3-sp15-modify-oracles-core-unit-budget/8047)
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md:29]: links need fixing

File: governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+* [MIP41c4-SP13: Facilitator Onboarding, Oracles Core Unit](https://github.com/makerdao/mips/blob/1a462c77d30bc603131b4f3c54a7486801395a7e/MIP39/MIP39c2-Subproposals/MIP39c2-SP3.md) | [Discussion Link](https://forum.makerdao.com/t/mip41c4-sp13-facilitator-onboarding-oracles-core-unit/8046)
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md:30]: and here too

File: governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+The sentence summary and paragraph summary have been provided by the MIP author:
+
+**Sentence Summary**
+
+*MIP39c2-SP13 adds the Oracles Core Unit.*
+
+**Paragraph Summary**
+
+*($paragraph_summary)*
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md:40]: I'd actually delete the missing paragraph summary if we don't hear back from Nik, and instead include the three sentence summaries:

The sentence summaries have been provided by the MIP author:

**Sentence Summary**

*MIP39c2-SP13 adds the Oracles Core Unit.*

*MIP40c3-SP15 adds the budget for the Oracles Core Unit.*

*MIP41c4-SP13 onboards Niklas Kunkel as Facilitator for the Oracle Core Unit.*

File: governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+Please review the linked discussion threads to inform your position on these proposals before voting. It was also mentioned and possibly discussed on the [Governance and Risk call](https://forum.makerdao.com/t/agenda-discussion-scientific-governance-and-risk-145-thursday-june-10-17-00-utc/8590) on Thursday, June 10.
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md:42]: "It was also mentioned[...]" -> "They were also mentioned[...]"

File: governance/polls/Ratification Poll - Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+## Review
+
+The community can vote in this poll to express support or opposition to this proposal being accepted and implemented in the Maker Protocol.
+
+The proposal in its entirety [can be found here](https://github.com/makerdao/mips/blob/40de9327bb3df265f6074c32710b68f2b5bb681e/MIP40/MIP40c3-Subproposals/MIP40c3-SP17.md). The sentence summary and paragraph summary have been provided by the MIP author:
+
+**Sentence Summary**
+
+*MIP40c3-SP13 adds the budget for Core Unit RISK-001: Risk.*
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021.md:30]: Should be: 

MIP40c3-SP17 adds the MKR Incentive Plan budget for Core Unit SES-001: Sustainable Ecosystem Scaling.

File: governance/polls/Ratification Poll - Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+
+**Paragraph Summary**
+
+_MIP40c3-SP17 adds the MKR Incentive Plan budget for Core Unit SES-001: Sustainable Ecosystem Scaling. It contains:_
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021.md:34]: Should be : 

MIP40c3-SP17 adds the MKR Incentive Plan budget for Core Unit SES-001: Sustainable Ecosystem Scaling. It contains:

- Total MKR Expenditure Cap
- Estimated MKR Expenditure (based on the current team)
- Escrow Wallet mechanism

MKR incentives have been determined based on the [Program discussed here](https://forum.makerdao.com/t/pre-mip-discussion-an-alternative-mkr-compensation-plan/8000). This is a 3-year vesting plan with 1-year cliff vest.

File: governance/polls/Ratification Poll - Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+
+- _Total MKR Expenditure Cap_
+- _Estimated MKR Expenditure (based on the current team)_
+- _Escrow Wallet mechanism_
+
+_MKR incentives have been determined based on the [Program discussed here](https://forum.makerdao.com/t/pre-mip-discussion-an-alternative-mkr-compensation-plan/8000). This is a 3-year vesting plan with 1-year cliff vest._
+
+Please review the discussion [thread](https://forum.makerdao.com/t/mip40c3-sp13-modify-core-unit-budget-risk-001/8034) to inform your position on this proposal before voting. It was also mentioned and possibly discussed on the [Governance and Risk call](https://forum.makerdao.com/t/agenda-discussion-scientific-governance-and-risk-145-thursday-june-10-17-00-utc/8590) on Thursday, June 10.
```
Comment (0xElihu) [inline governance/polls/Ratification Poll - Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021.md:42]: Wrong thread link. Should be: https://forum.makerdao.com/t/mip40c3-sp17-sustainable-ecosystem-scaling-core-unit-mkr-budget-ses-001/8043

**Review by 0xElihu — CHANGES_REQUESTED** [review]

File: governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md (hunk 3 above)
Comment (blimpa) [reply governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md:4]: We're using the mandate for the time being

File: governance/polls/Ratification Poll - Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021.md (hunk 10 above)
Comment (blimpa) [reply governance/polls/Ratification Poll - Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021.md:34]: I don't see anything wrong with this one

**Pull request merged**